vite.config.ts

```diff
@@ -36,6 +36,7 @@ export default defineConfig( {
 				__dirname,
 				'node_modules/@wmde/wikit-vue-components/dist/wikit-vue-components.css',
 			),
+			'@wmde/wikit-vue-components/src/styles/mixins/Typography': '@wmde/wikit-vue-components/src/styles/mixins/Typography',
```

Comment:
Hm, does that mean the previous alias also doesn’t need the `resolve()` call?

Reply:
Yeah, maybe :thinking:

Reply:
#49